source/index.js

```diff
@@ -15,7 +15,8 @@ const pkgDir = require('pkg-dir');
 const hostedGitInfo = require('hosted-git-info');
 const prerequisiteTasks = require('./prerequisite-tasks');
 const gitTasks = require('./git-tasks');
-const publishTaskHelper = require('./publish-task-helper');
+const publish = require('./npm/publish');
```

Comment:
I could merge these two into

```js
const {publish, enable2fa} = require('./npm');
```

I don't have a strong opinion, this might be a bit cleaner.